test/test_bot.py

```diff
@@ -661,8 +661,7 @@ def testrule(bot, trigger):
         plugin='testplugin',
         label='testrule',
         handler=testrule,
-        rate_limit=100,
-        threaded=False,
```

Comment:
Why's this deleted? It came along recently, in a2a8ba0fb65fedfc39b7b7b9d639ba90a5a740df (from #2297), so it's probably important.

Reply:
Oh! It's because I messed up in #2297 at some point in my development process and it wasn't necessary at all, but I forgot to remove it. It's the equivalent of ` print('debug')` but very sneaky.